### DISCUSSION TITLE
Green borders shortly shows up when GUI is refreshed

### DISCUSSION THREAD

**Discussion by AdagioB** [quasarframework/quasar#13291] [General - Components / Directives / etc] [answered]
In the last few days, I keep getting green lines (as shown in the gif below), whenever I load the page or data changes causing the components to change
Currently this is only happening on my development machine as nothing has been released yet. This is using the newest Vue 3 and newest Quasar. So far I have only seen this in Chrome, the browser I'm mostly using during development (and the browser of most our users)
It helps to restart Chrome, but after a short while it comes back. I haven't made any changes in the code that should cause this

Any idea on what could be the cause? I can't tell if this is Quasar or Vue related

<a href="https://imgur.com/uXxD1Ym"><img src="https://i.imgur.com/uXxD1Ym.gif" title="source: imgur.com" /></a>

**Comment by MilosPaunovic** [comment]
It might be some option from VueDevtools, give that a check for start.

**Reply by AdagioB** [reply]
Thank you, there seems to be an option there I might have enabled a few days ago, that caused this

**Comment by pdanpdan** [comment, marked as answer]
You have enabled `Highlight updates` option in VueDevTools (components tab, ... menu on right)

**Reply by AdagioB** [reply]
Looks like it might be that. At first it didn't work, but after restarting the PC it worked (at least so far)
Thank you for the help
